vertx-web-client/src/main/java/io/vertx/ext/web/client/WebClientSession.java

```diff
@@ -65,6 +69,7 @@ static WebClientSession create(WebClient webClient, CookieStore cookieStore) {
    * @return a reference to this, so the API can be used fluently
    */
   @Fluent
+  @GenIgnore(GenIgnore.PERMITTED_TYPE)
```

Comment:
We would need to check what happens in the generators. If all the methods are skipped because of GenIgnore, the benefits would be quite diminished.

Reply:
I had to tag almost all of the methods as `GenIgnore` because they use `CharSequence` and `Iterator` and those are not "permitted types". If I just do `GenIgnore` then there's almost no methods.

I can do it but I guess we need to consider allowing `Iterator` on codegen as it would simplify some APIs